modules/config_import.py

```diff
@@ -1,13 +1,54 @@
-def get_config():
+def get_language(language, platform):
     #Imports
     import os
     from configparser import ConfigParser
     from configparser import ExtendedInterpolation
     import json
+    print(os.getcwd())
+    #Get Config
+    conf = {}
+    os.chdir('config/languages')
+    config = ConfigParser(interpolation=ExtendedInterpolation())
+    config.read(language + '_return.ini')
+    if platform == 'windows': #Windows
+        conf['err_0'] = config['Return_Dos']['err_0']
+        conf['err_1'] = config['Return_Dos']['err_1']
+        conf['err_2'] = config['Return_Dos']['err_2']
+        conf['err_3'] = config['Return_Dos']['err_3']
+        conf['err_5'] = config['Return_Dos']['err_5']
+        conf['err_8'] = config['Return_Dos']['err_8']
+        conf['err_12'] = config['Return_Dos']['err_12']
+        conf['err_14'] = config['Return_Dos']['err_14']
+        conf['err_19'] = config['Return_Dos']['err_19']
+        conf['err_54'] = config['Return_Dos']['err_54']
+        conf['err_65'] = config['Return_Dos']['err_65']
+        conf['err_82'] = config['Return_Dos']['err_82']
+        conf['err_148'] = config['Return_Dos']['err_148']
+        conf['err_266'] = config['Return_Dos']['err_266']
+    else: #Linux or MacOS
+        conf['err_0'] = config['Return_Unix']['err_0']
```

Comment:
Same comment of the **if** above:
```suggestion
        unix_return = 'Return_Dos'
        for key in conf.keys():
            conf[key] = config[unix_return][key]
```